content/guides/references/migration-guide.md

```diff
@@ -2,14 +2,256 @@
 title: Migration Guide
 ---
 
-<DocsVideo src="https://youtube.com/embed/mIqKNhLlPcU"></DocsVideo>
+## Migrating to Cypress version 11.0
+
+This guide details the changes and how to change your code to migrate to Cypress
+version 11.0.
+[See the full changelog for version 10.0](/guides/references/changelog#11-0-0).
+
+### Component Testing Changes
```

Comment:
This is the second `Component Testing Changes` section within this page, which causes the latter to be incremented to make it unique e.g. https://cypress-documentation-gdkinysy2-cypress-io.vercel.app/guides/references/migration-guide#Component-Testing-Changes-1

Doesn't look like we have an on-link attached to this and seems like an edge-case, just wanted to point it out

Reply:
I noticed this as well, I actually pushed a change to your Services PR changing this to "Component Testing Updates" to make it unique.